cpu/esp_common/esp-now/esp_now_gnrc.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2018 Timo Rothenpieler
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     cpu_esp_common_esp_now
```

Comment:
Travis points this out, but since https://github.com/RIOT-OS/RIOT/pull/10668 is merged now this shouldn't be a problem.

Reply:
Rebased again, so this shouldn't be an issue anymore.